src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpoint.java

```diff
@@ -56,6 +63,9 @@ public CaseAccessEndpoint(CaseAccessOperation caseAccessOperation) {
         @RequestParam(value = "userId") final String idSearchingFor
     ) {
         LOG.debug("Finding cases user: {} has access to", idSearchingFor);
+        if (applicationParams.getEnableAttributeBasedAccessControl()) {
+            return roleAssignmentService.getCaseIdsForAGivenUser(idSearchingFor);
+        }
```

Comment:
any cache required for roleAssignmentService#gets(..)  as current one has request scope in CachedCaseUserRepository ?

Reply:
This is going to be implemented in another ticket.  Rafal is working with it